docs/csharp/write-safe-efficient-code.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Write safe and efficient code
+description: Recent enhancements to the C# language enable you to write verifiable safe code that the performance previously associated with unsafe code. 
+ms.date: 11/16/2017
+ms.custom: mvc
```

Comment:
is this is mvc, what is the topic type? I thought conceptual topics were not considered part of mvc, but I could be wrong.

Reply:
Yes, conceptual is a topic type.